### DISCUSSION TITLE
Wifi pineapple tetra

### DISCUSSION THREAD

**Discussion by ghost** [xchwarze/wifi-pineapple-cloner#19] [Q&A]
Forgive me if this is not the right place to post and forgive me if i seem new to this stuff, the firmware on patreon for the tetra seems not to go through with sysupgrade or firmware recovery options i tried both methods and also tried the tetra .bin and .tar. the nano firmware works a treat just having issues with the tetra firmware. if you need more info let me know :) edit, like it goes through but the tetra light stays yellow.

**Comment by xchwarze** [comment]
hak5 never released the patches to be able to compile and upgrade that hardware, so it cannot be upgraded.
I tried to make a build with what I found but it never ends up working.

**Reply by ghost** [reply]
ok i understand :) thank you for the reply and all the hard work you have done so far with every build.

**Comment by ghost** [comment]
Edit: sorry lol ummm I like the tetra better then the nano and at 139 for the new pineapple too much for my blood to afford I don't want it to be a paper weight, what else can be done to mod the tetra to give it life? I heard your working on a project that works like the pineapple but has none of there code wish there was a place to learn more about it. Last question I have the gl.inet mango v2 and loaded your prebuilt cloner firmware works good just how can I add two wireless cards to this little thing lol ?I was also bored yesterday and was like what if I load the pineapple Vll firmware recovery to the mango all went well but when I press the reset button nothing happens so that was a bust I know the mango does not support the new pineapple firmware i was just bored lol. Sorry in advanced for adding to this section :(

**Comment by xchwarze** [comment]
As I commented in other threads the reality is that without passing the two or three hard checks that the mk7 has, nothing can be done. Also they changed the whole license of the product, I guess to be able to better chase anyone who tries to reverse....

That's why I decided to make my own gpl platform and whoever wants can make modules and practice with it.

**Comment by ghost** [comment]
Well said thank you :) your platform is going to be very cool :)
